src/main/scala/cromwell/util/FileUtil.scala

```diff
@@ -10,6 +12,14 @@ object FileUtil {
     File.createTempFile(baseName, ".tmp", directory).toPath.fileAndWriter
   }
 
+  def parseTsv(tsv: String): Try[Array[Array[String]]] = {
+    val table = tsv.split("\n").map {row => row.split("\t")}
+    table.map(_.size).toSet match {
```

Comment:
Whichever of the two you choose above, it'd be nice if you did the same one here